### PR TITLE
[Polkadot] Can only nominate only if account has minimum bounded DOT

### PR DESCRIPTION
**Pull request by lvndry**
### 📝 Description

Update condition so that an account can only nominate polkadot if they have enough bounded DOT

### ❓ Context

- **Impacted projects**: `LLD` `LLM` `LLC`
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-2355

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo
<details>
<summary>LLD</summary>
Screenshot 2023-10-27 at 15 17 17

Screenshot 2023-10-27 at 15 17 31

Screenshot 2023-10-27 at 15 17 42
</details>

<details>
<summary>LLM</summary>

simulator_screenshot_8C63100D-7A3A-4C50-AE79-FF38F6C46C32

simulator_screenshot_9B5BEF73-FB7C-42AB-A1CF-EB0A68B738B9

simulator_screenshot_4A7ED2B9-00E7-42E3-8425-8146CE6D14CB

</details>

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/static/i18n/en/app.json [hunk 1]
```diff
@@ -3158,7 +3158,7 @@
     "unlockedBalance": "Unbonded",
     "unlockedTooltip": "Unbonded assets can now be moved using the withdraw operation.",
     "networkFees": "Network fees are automatically set by the Polkadot consensus, you won't be able to review them on your device",
-    "bondedBalanceBelowMinimum": "Your bonded balance is below the current minimum of {{minimumBondBalance}}. Your nominations are at risk of being removed.",
+    "bondedBalanceBelowMinimum": "As a nominator, a minimum of {{minimumBondBalance}} is required to submit an intention to nominate",
```
Comment (hedi-edelbloute) [inline apps/ledger-live-desktop/static/i18n/en/app.json:3161]: Your bonded balance is below the current minimum of {{minimumBondBalance}}. You cannot submit nomination requests.

Thoughts ?

File: apps/ledger-live-mobile/src/locales/en/common.json [hunk 2]
```diff
@@ -5375,7 +5375,7 @@
     "unlockingBalance": "Unbonding balance",
     "unlockedBalance": "Unbonded balance",
     "networkFees": "Network fees are automatically set by the Polkadot consensus, you won't be able to review them on your device",
-    "bondedBalanceBelowMinimum": "Your bonded balance is below the current minimum of {{minimumBondBalance}}. Your nominations are at risk of being removed.",
+    "bondedBalanceBelowMinimum": "As a nominator, a minimum of {{minimumBondBalance}} is required to submit an intention to nominate",
```
Comment (hedi-edelbloute) [inline apps/ledger-live-mobile/src/locales/en/common.json:5378]: Shouldn't this be minBondWarning's description being edited ?

File: apps/ledger-live-mobile/src/families/polkadot/AccountBalanceSummaryFooter.tsx [hunk 3]
```diff
@@ -141,7 +154,9 @@ function useInfo(): Record<InfoName, ModalInfo[]> {
       {
         Icon: () => <WarningIcon color={colors.orange} size={18} />,
         title: t("polkadot.info.minBondWarning.title"),
-        description: t("polkadot.info.minBondWarning.description"),
+        description: t("polkadot.bondedBalanceBelowMinimum", {
```
Comment (hedi-edelbloute) [inline apps/ledger-live-mobile/src/families/polkadot/AccountBalanceSummaryFooter.tsx:157]: see my comment on translations

**Review by hedi-edelbloute — APPROVED** [review]

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**